aws-rotate-keys/orb.yml

```diff
@@ -30,6 +30,10 @@ commands:
           you will use to hold this value, i.e. AWS_SECRET_ACCESS_KEY.
         default: AWS_SECRET_ACCESS_KEY
     steps:
+      - aws-cli/setup:
```

Comment:
Is this potentially going to break things for those who don't use an executor and already have aws cli installed? Would it make sense to only run this step if aws cli isn't found on the path (i.e. running this orb using the default executor)?

Reply:
this step checks if aws-cli is already installed before trying to install it

next, it uses the aws-cli from path to configure the credentials